auth_matrix/templates/auth_matrix/admin/authorization_matrix.html

```diff
@@ -11,7 +11,8 @@
     .matrix-table th,
     .matrix-table td {
     border: 1px solid black;
-    padding: 8px;
+    padding: 10px;
+    font-size: 14px;
```

Comment:
so the purpose of inheriting django templates was that we ideally could get rid of all custom CSS - I see that you haven't succeeded with that yet. Shall we spend some time this week to figure this out?

Reply:
de-fi-nite-ly !